cmd/process-agent/config.go

```diff
@@ -66,21 +66,29 @@ func getChecks(sysCfg *sysconfig.Config, oCfg *oconfig.OrchestratorConfig, canAc
 }
 
 func getAPIEndpoints() (eps []apicfg.Endpoint, err error) {
+	return getAPIEndpointsWithKeys("https://process.", "process_config.process_dd_url", "process_config.additional_endpoints")
+}
+
+func getEventsAPIEndpoints() (eps []apicfg.Endpoint, err error) {
+	return getAPIEndpointsWithKeys("https://process-events.", "process_config.events_dd_url", "process_config.events_additional_endpoints")
+}
+
+func getAPIEndpointsWithKeys(prefix, defaultEpKey, additionalEpsKey string) (eps []apicfg.Endpoint, err error) {
```

Comment:
What does`Ep` and `Eps` stands for in `defaultEpKey` and `additionalEpsKey`, I'm not sure

Reply:
They stand for endpoint. What abbreviation do you think we should use?

Reply:
We're using Ep in other parts of the code as well that's why I ended up picking it. Let me know if you think that we should use a better acronym. I can update it in a follow-up PR.
https://github.com/DataDog/datadog-agent/blob/65854dd66094dd278aa5ae0bbd6f44b6576ca079/cmd/process-agent/collector.go#L295-L302